api/src/services/db/providerStatusService.ts

```diff
@@ -42,6 +42,7 @@ export async function getNetworkCapacity() {
 }
 
 export const getProviderList = async () => {
+  // Fetch provider list with their attributes & auditors
```

Comment:
imho this kind of comments don't add help much. It's already obvious that `providers` is a providers list. If it's essential to emphasise it's got attributes and auditors (is it?) it can be named accordingly. e.g. `providersWithAttrsAndAuditors`. It would also help understanding code better further if it's important.